website/content/en/docs/building-operators/ansible/reference/advanced_options.md

```diff
@@ -263,7 +263,29 @@ Using the `--ansible-log-events` CLI flag, you can determine to what degree the
 2. `Tasks` - Only Ansible Tasks will be outputted.
 3. `Everything` - All info logs and all tasks will be outputted.
 
-If you want more control over the logs that are outputted, consider using the [Zap Logger][Zap-Logger] and [verbosity annotations][verbosity-annotations] in tandem with the `--ansible-log-events` CLI flag. 
+If you want more control over the logs that are outputted, consider using the [Zap Logger][Zap-Logger] and [verbosity annotations][verbosity-annotations] in tandem with the `--ansible-log-events` CLI flag.
+
+## `ansible.sdk.operatorframework.io/reconcile-period` Custom Resource Annoation
+
+While running a Ansible-based operator, the reconcile-period can be specified through the custom resource's annotations under the `ansible.sdk.operatorframework.io/reconcile-period` key. 
```

Comment:
```suggestion
You can specify the reconcile period for an Ansible Operator by editing the `ansible.sdk.operatorframework.io/reconcile-period` key in the custom resource annotation file. 
```
Would it help the user to have the full filepath here? Like: "... in the `watches.yaml` custom resource annotation file."

Reply:
"Editing" suggests that this field is already present in the custom resource when the operator gets scaffolded. However, a user would have to manually add this field to use this feature.

I think mentioning custom resource is sufficient since it is a major component of the operator and authors usually know which file (YAML formatted) this corresponds to. Annotations are a subsection of this file.

Reply:
WDYT of the following:
"You can specify the reconcile period for an Ansible Operator by adding the `ansible.sdk.operatorframework.io/reconcile-period` key to the custom resource annotations."